src/chrome/content/rules/Eclipse.xml

```diff
@@ -1,22 +1,21 @@
 <!--
 	Nonfunctional domains:
 
-		wiki.eclipse.org	(redirects to marketplace 404)
+		download.eclipse.org
 
 -->
 <ruleset name="Eclipse (partial)">
 
 	<target host="eclipse.org" />
-	<target host="*.eclipse.org" />
-
+	<target host="bugs.eclipse.org" />
+	<target host="dev.eclipse.org" />
+	<target host="git.eclipse.org" />
+	<target host="www.eclipse.org" />
+	<target host="wiki.eclipse.org" />
+	<target host="marketplace.eclipse.org" />
 
 	<securecookie host="^bugs\.eclipse\.org$" name=".*" />
 
-
-	<rule from="^http://((?:bugs|dev|git|www)\.)?eclipse\.org/"
-		to="https://$1eclipse.org/" />
-
-	<rule from="^http://marketplace(1)?\.eclipse\.org/(eclipse\.org-common|misc|modules|sites)/"
```

Comment:
It looks like this rule used to only do HTTPS on certain paths on marketplace. Have you checked some other paths on the site and made sure they support HTTPS properly?